### ISSUE TITLE
1.11 - warp not showing list of saved warps

### ISSUE THREAD

**Issue by smmmadden** [EssentialsX/Essentials#980]
I've migrated from using Essentials-2x to EssentialsX but retained all my warp .yml files.  Unfortunately, using /warp xxxxx does not find the valid warp location and /warp doesn't show the list of valid warps.  I created a new warp location to see if the format/structure is different and while the warp name was created and I can use /warp myLocal, I still can't get a list of warps and it didn't create the yml file for that warp.

Summary:
1. /warp - listing not returned, instead "Too few arguments. /warp [-s] world [target] <warp>
2. /warp myLocal - works if it is created, but does not create associated yml file in Essentials\warp folder
3. /setwarp myLocal works, but not saved and not found in listing

Environment: Spigot 1.11 on Windows 10 Pro x64
![warp-issue](https://cloud.githubusercontent.com/assets/10309346/20549784/75c035b2-b0fd-11e6-9f76-5eede60cf761.jpg)

**Comment by smmmadden** [comment]
The same warp points work fine when used on a Sign 
[Warp]
MyLocal
So appears to be command-line only

**Comment by mdcfe** [comment]
Another plugin is overriding /warp. Could you paste a list of plugins?

**Comment by smmmadden** [comment]
[07:03:04 INFO]: Plugins (29): WorldEdit 6.1.5, TitleMotd 2.1, OpenInv 2.3.9-vk2gpz-0.1, GroupManager 1.9, PlaceholderAPI 2.6.0, MythicLib 1.0.1, ProtocolLib 4.1.0, EditableSign 2.4.1, HorseTpWithMe 1.4.11, LagMonitor 1.11.3, Vault 1.5.6-b49, CommandBook 2.4:1692-75bb1c8,master, dynmap 2.3, Essentials 2.0.1, WorldBorder 1.8.6, HeadDatabase 2.3.2, LWC 1.7.3, Fe 0.8.3, MythicMobs 2.5.1, Residence 4.5.12.1, EssentialsProtect 2.0.1, EssentialsChat 2.0.1, EssentialsGeoIP 2.0.1, EssentialsAntiBuild 2.0.1, CraftBook 3.9.12, EssentialsSpawn 2.0.1, SelectionVisualizer 1.6.4.0, ChestShop 3.8.12, Jobs 3.7.1

**Comment by mdcfe** [comment]
Why are you using both CommandBook and Essentials? CommandBook looks like the most likely culprit for this.

**Comment by smmmadden** [comment]
good question, though CommandBook does show warp as being enabled and it was working in the previous Essentials version it looks like users only need one or the other, but not both.  

I don't recall why I had added CommandBook (2 years ago) & Essentials when they appear to be sharing the same features.  I've removed CommandBook and restarted and now warp is working perfectly with the list and use.  I'll need to do some digging on what was different about the two that made me install both.  But this issue can be closed.  Thanks again!
